CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+🏥 We're glad you want to help!
+
+## Goals of the project
+
+*  *Stay lightweight*: depend on as much of the standard library as possible; minimize external dependencies.
+* *Self-serving*: `attending` should manage its own docs; add in a bootstrap option.
+* *Share a cache*: Multiple environments can use the same files.
+
+## Get the code
+
+1. Fork this repository.
+2. `git clone` the forked repository, and `cd` to it.
+3. Get the development libraries with `pip install -r requirements-dev.txt`
+
+## Testing your changes
+
+Just run `tox`.
```

Comment:
"Just" ends up being a barrier for folks who have no idea what `tox` is. Can we add a tox link, and remove the "just"?